doc/api/cli.md

```diff
@@ -127,6 +127,15 @@ added: v12.0.0
 
 Specify the file name of the CPU profile generated by `--cpu-prof`.
 
+### `--disable-proto=mode`
+<!--YAML
+added: REPLACEME
+-->
+
+Disable the `Object.prototype.__proto__` property. If `mode` is `delete`, the
+property will be removed entirely. If `mode` is `throw`, accesses to the
+property will throw an exception with the code `ERR_PROTO_ACCESS`.
```

Comment:
Clarify how worker threads and vm contexts are affected:

```suggestion
Disable the `Object.prototype.__proto__` property in the main global contexts
including worker threads; it will not be disabled in `vm` created contexts. If
`mode` is `delete`, the property will be removed entirely. If `mode` is `throw`,
accesses to the property will throw an exception with the code
`ERR_PROTO_ACCESS`.
```